pkgs/top-level/all-packages.nix

```diff
@@ -19076,10 +19076,10 @@ with pkgs;
 
   captive-browser = callPackage ../applications/networking/browsers/captive-browser { };
 
-  catboost = callPackage ../development/libraries/catboost {
-    # https://github.com/catboost/catboost/issues/2540
-    cudaPackages = cudaPackages_11;
-  };
+  # catboost = callPackage ../development/libraries/catboost {
+  #   # https://github.com/catboost/catboost/issues/2540
+  #   cudaPackages = cudaPackages_11;
+  # };
```

Comment:
Do these need to be here still, or can be removed entirely?

Reply:
I was testing if it was possible to remove it. But unfortunately no, we have to pin cuda to version 11.